.travis.yml

```diff
@@ -4,9 +4,6 @@ matrix:
   allow_failures:
 
 rvm:
-  - 2.1.10
-  - 2.2.7
-  - 2.3.4
```

Comment:
Drive-by comment: do we have a policy around deprecating major ruby versions in open source projects? I recall seeing some discussions around this a while ago (for statesman I think)

Reply:
Unsure of any policy we may have. I'm pretty sure that these versions never worked. @jaysh can confirm

Reply:
@ivgiuliani We should double check the next time we plan on deprecating (to be clear, this PR is not actually deprecating anything - it's already broken).

In this instance, the failure here was that the code doesn't work at all with Ruby < 2.4  and we weren't observing our builds so this went unnoticed. Now that we have visible build results in the PRs, this shouldn't happen going forwards. That said, if you feel there's value in getting the project working with Ruby 2.1+ again, I can put a ticket in for it.

Reply:
For reference, this is what I was referring to: https://github.com/gocardless/statesman/pull/253
Not saying we should introduce the same policy here (though in this case it's simpler since we only care about the ruby version, not rails too), but worth thinking about it.